.idea/gradle.xml

```diff
@@ -7,15 +7,14 @@
         <option name="testRunner" value="PLATFORM" />
         <option name="distributionType" value="DEFAULT_WRAPPED" />
         <option name="externalProjectPath" value="$PROJECT_DIR$" />
-        <option name="gradleJvm" value="1.8" />
+        <option name="gradleHome" value="/usr/local/Cellar/gradle/7.0.1/libexec" />
```

Comment:
🤔  maybe  I shouldn't commit this :D 

it might break your builds, so let me know!